policyengine_us/variables/gov/states/mi/tax/income/deductions/mi_retirement_benefits_deduction_tier_three.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class mi_retirement_benefits_deduction_tier_three(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan standard deduction"
+    unit = USD
+    definition_period = YEAR
+    documentation = "Michigan retirement and pension benefits for age 62-66."
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-30",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf?rev=86a928564e3f42449c531309673f5da7&hash=7147C48E7C9B1B8171B72DC34A66642A",
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.deductions.standard.retirement_benefits_deduction.tier_three
+        # Core deduction based on filing status.
+        filing_status = tax_unit("filing_status", period)
+
+        age_older = max_(
```

Comment:
Let's make a new variable in the same place as `age_head` and `age_spouse` called `greater_age_head_spouse` to avoid duplicating this logic.

Reply:
should I keep the inputs in test file the same as `age_head` and `age_spouse`, or just 1 `greater_age_head_spouse`?

Reply:
Unit tests should match the variable formula, so if you change the variable inputs to `greater_age_head_spouse`, please do the same in the test